apps/sso/components/login-card/login-card.tsx

```diff
@@ -137,3 +137,22 @@ const StyledErrorText = styled(Text)`
   padding-top: 1rem;
   color: red;
 `;
+const Input = styled.input`
```

Comment:
nit: I noticed that the new Input components across all the files are the same; you should extract it out into its own component and use the same one for all files instead.